dotcom-rendering/src/experiments/tests/usa-expandable-marketing-card.ts

```diff
@@ -7,7 +7,7 @@ export const UsaExpandableMarketingCard: ABTest = {
 	author: 'dotcom.platform@guardian.co.uk',
 	description:
 		'Test the impact of showing the user a component that highlights the Guardians journalism.',
-	audience: 15 / 100,
+	audience: 40 / 100,
```

Comment:
The audience is equally split between the three variants and the control. 10% in each group